include/libs/qcom/query.h

```diff
@@ -207,6 +207,12 @@ typedef struct SQueryNodeStat {
   int32_t tableNum;  // vg table number, unit is TSDB_TABLE_NUM_UNIT
 } SQueryNodeStat;
 
+typedef struct SColLocation {
```

Comment:
If we can use the SColumn, instead of whole new struct, as it is not used to transfer message?